services/elasticloadbalancing/src/it/java/software/amazon/awssdk/services/elasticloadbalancing/ElbIntegrationTest.java

```diff
@@ -483,7 +483,7 @@ public void testSetLoadBalancerPoliciesOfListener() {
         assertNotNull(policyDescriptions.get(0).policyTypeName());
 
         // Remove the policy from the listener
-        elb.setLoadBalancerPoliciesOfListener(SetLoadBalancerPoliciesOfListenerRequest.builder().loadBalancerName(loadBalancerName).loadBalancerPort(80).policyNames().build());
+        elb.setLoadBalancerPoliciesOfListener(SetLoadBalancerPoliciesOfListenerRequest.builder().loadBalancerName(loadBalancerName).loadBalancerPort(80).build());
```

Comment:
I prefer the previous behavior where you had to set an explicitly empty list to remove policies. This feels a bit safer and per the documentation this is a required member anyways so it lines up with the service API better. We should treat auto construct lists as effectively null.

https://docs.aws.amazon.com/elasticloadbalancing/2012-06-01/APIReference/API_SetLoadBalancerPoliciesOfListener.html